blogposts/2021/optimizing-a-code-intelligence-commit-graph-2.md

```diff
@@ -0,0 +1,494 @@
+---
+title: "Optimizing a code intelligence commit graph (Part 2)"
+author: Eric Fritz
+authorUrl: https://eric-fritz.com
+publishDate: 2021-05-12T18:00-07:00
+tags: [blog]
+slug: optimizing-a-code-intel-commit-graph-part-2
+heroImage: https://sourcegraphstatic.com/blog/lsif-go/lsif-go-improvements.png
+published: true
```

Comment:
```suggestion
published: false
```

Let's hold off publishing part 2 for now so we can go ahead with part 1 tomorrow (May 25) and finesse part 2.